src/api.v2/helpers/worker/workSubmit/submitWorkForHook.js

```diff
@@ -0,0 +1,33 @@
+const validateAndSubmitWork = require('../../../events/validateAndSubmitWork');
+const generateETag = require('../generateEtag');
+
+
+const submitWorkForHook = async (experimentId, authJWT, body) => {
+  const extras = undefined;
+
+  const data = {
+    experimentId,
+    body,
+    extras,
```

Comment:
if `extras` is always undefined, why would we not remove it completely?

Reply:
because this is mimicking the behaviour from the UI calls. It needs the same parameters to produce the same ETag